C4SGSvc/src/main/resources/application.properties

```diff
@@ -5,7 +5,7 @@
 # Set here configurations for the database connection
 
 # Connection url for the database "netgloo_blog"
-spring.datasource.url = jdbc:mysql://localhost:3306/C4SG
+spring.datasource.url = jdbc:mysql://localhost:3306/c4sg
```

Comment:
this change will force devs to create a new schema right? i ran into this error when i setup, where i had to create 'C4SG' and not 'c4sg'